site/content/posts/2021-06-10-docs-wg.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Contour Technical Documentation Working Group Kick Off
+excerpt: Announcing Contour Technical Documentation Working Group
+author_name: Contour Team
+author_avatar: /img/contributors/orlin-vasilev.jpg
+categories: [kubernetes]
+tags: ['Contour Team', 'Abigail McCarthy', 'Docs' ,'Docs-WG', 'Jonas Rosland', 'Orlin Vasilev']
+date: 2021-06-10
+slug: docs-wg
+---
+
+
+
+# Join the Contour Tech Docs Working Group
+
+The Contour team is excited to announce we are launching a technical documentation working group. This group will focus on improving the Contour website, documentation, and processes around content creation.
+
+# Because documentation makes a difference
+
+In 2017, GitHub conducted the Open Source Survey, which sampled 5,500 respondents. One of the key takeaways was that “documentation is highly valued, but often overlooked.” The survey found that outdated or confusing documentation was a major problem, affecting 93% of respondents. Yet 60% of contributors rarely or never contribute to documentation. We want this working group to be an opportunity to give back to the community by helping to improve the docs.
+
+# The purpose and scope of the Working Group
+
+The purpose of the Working Group is to build up a self-sustaining community around documentation for Contour. We have an initial need to rework the existing documentation based on the recommendations outlined in the CNCF Tech Docs Review, and a continuous need for documentation being added/edited/removed for releases going forward.
```

Comment:
Missing link to the CNCF Tech Docs Review: https://github.com/cncf/techdocs/blob/main/assessments/0001-contour.md

Reply:
fixed